README.md

```diff
@@ -1,2 +1,90 @@
-# CIROH-open-source-project-template
-CIROH open source project template 
+# CIROH Open Source Project Template Instructions
+
+1. Start a new project.
+2. [Copy these files into your new project](#installation).
+3. Update the README, replacing the contents below as directed.
+4. Include any libraries, assets, or hard dependencies whose source code will be included in the project's repository in the _Exceptions_ section of the [TERMS](TERMS.md).
+   - If no exceptions are needed, you can remove that section from TERMS.
+5. If you're working with an existing codebase, answer the questions on the [open source checklist](opensource-checklist.md).
+6. Delete these instructions and everything up to the _Project Title_ from the README.
+7. Get to work and make something awesome!
+
+> Keep the README up-to-date! It's often the first thing people see and forms their initial impression.
+
+## Installation
+
+To install all of the template files, run the following script from the root of your project's directory:
+
+```
+bash -c "$(curl -s https://raw.githubusercontent.com/sepehrkrz/CIROH-open-source-project-template/fe745bff03da976202b376b62cdb42d7de2f92de/open_source_template.sh)"
+
+```
+
+----
+
+# Project Title
+
+**Description**: 
+Provide a concise, plain-language description of your project's purpose, its significance, and the problems it addresses. Explain how this software can benefit its intended audience.
+
+Additional details to include:
+
+- **Technology Stack**: Specify the primary programming language(s) and whether the software is standalone or part of a larger ecosystem.
+- **Status**: Indicate the current development stage (e.g., Alpha, Beta, Version 1.0). You may also link to the [CHANGELOG](CHANGELOG.md) for version history.
+- **Links to Production or Demo Instances**: If applicable, provide links to live deployments or demo environments.
+- Highlight what sets your project apart from similar ones. If necessary, link to additional documentation or pages for further explanation.
+
+**Screenshot**: 
+If your project has a visual component, consider adding a screenshot here:
+
+![](https://raw.githubusercontent.com/sepehrkrz/CIROH-open-source-project-template/fd0c726ccae13b2196cc5f1bb54b763400e1634e/doc/Screenshot.png
```

Comment:
Shouldn't this direct to 
```
![](doc/Screenshot.png)
```
 instead of the github content link? I notice this is exactly what's done in other repositories, but it seems inelegant if you can do a local / relative link instead.